packages/calcite-components/src/tests/commonTests.ts

```diff
@@ -194,7 +194,7 @@ function propToAttr(name: string): string {
 }
 
 /**
- * Helper for asserting that a property's value is its default
+ * Helper for asserting that a property's value is its default.
```

Comment:
Thanks for updating the doc here. 

Can this be submitted separately? There aren't any changes related to this PR, so it's out of scope.